sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -264,7 +269,22 @@ private void applySemanticConventions(Attributes attributes, RemoteDependencyDat
         }
         String messagingSystem = attributes.get(AttributeKey.stringKey("messaging.system"));
         if (messagingSystem != null) {
-            applyMessagingClientSpan(attributes, remoteDependencyData, messagingSystem, spanKind);
+            applyMessagingClientSpan(attributes, remoteDependencyData, messagingSystem, span.getKind());
+            return;
+        }
+        // TODO (trask) ideally EventHubs SDK should conform and fit the above path used for other messaging systems
+        //  but no rush as messaging semantic conventions may still change
+        //  https://github.com/Azure/azure-sdk-for-java/issues/21684
+        String name = span.getName();
+        if (name.equals("EventHubs.send") || name.equals("EventHubs.message")) {
+            applyEventHubsSpan(attributes, remoteDependencyData);
+            return;
+        }
+        // TODO (trask) ideally ServiceBus SDK should conform and fit the above path used for other messaging systems
+        //  but no rush as messaging semantic conventions may still change
+        //  https://github.com/Azure/azure-sdk-for-java/issues/21686
+        if (name.equals("ServiceBus.message") || name.equals("ServiceBus.process")) {
```

Comment:
`ServiceBus.process` is a consumer span and should be request telemetry. Perhaps this should be `ServiceBus.send`?